src/fromager/__main__.py

```diff
@@ -79,6 +80,11 @@
     help="control removal of working files when a build completes successfully",
 )
 @click.option("--variant", default="cpu", help="the build variant name")
+@click.option(
+    "-j",
+    "--jobs",
+    help="number of jobs available to run in parallel",
+)
```

Comment:
I think we need a `type` argument here to force the value to be converted to an integer.

```
$ tox -e cli -- -j nine bootstrap fromager
...
TypeError: '>' not supported between instances of 'str' and 'int'
```